src/routes/docs/contributing/translating/+page.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Translating Files
+---
+
+# Translating
```

Comment:
```suggestion
```

Reply:
The `#` header is used for the title on line 2.

Reply:
Done.